lux/processor/Compiler.py

```diff
@@ -82,7 +82,21 @@ def compile_intent(ldf: LuxDataFrame, _inferred_intent: List[Clause]) -> VisList
         vis_collection: list[lux.Vis]
                 vis list with compiled lux.Vis objects.
         """
-        if _inferred_intent:
+        valid_intent = _inferred_intent  # ensures intent is non-empty
```

Comment:
Same comments as above with this. Are these the same checks? Can we share these?

Reply:
same as above!